Sources/RedisKit/RedisConnectionPool.swift

```diff
@@ -17,13 +17,11 @@ private struct _PoolRedisClient {
 }
 
 extension _PoolRedisClient: RedisClient {
-    var logger: Logger {
-        self.pool.logger
-    }
+    var isConnected: Bool { true }
```

Comment:
@tanner0101 This is the only part of this PR that I'm not 100% comfortable with. I'm not sure what to do here.

Reply:
After talking with Tanner, we agreed that this is "ok" as the connection pool is technically always connected - but since I couldn't recall why I felt a compelling use case of adding it to the RedisClient protocol - I should revert the requirement change in RediStack in an upcoming Alpha / Beta release.

So we'll merge it as is for now, but before tagging RC.